FRW/UI/Component/AlertView/AlertView.swift

```diff
@@ -188,3 +256,30 @@ struct AlertView_Previews: PreviewProvider {
         AlertViewTestView()
     }
 }
+
+#Preview("Insufficient Storage") {
+    let customContentView: () -> some View = {
+        VStack(alignment: .center, spacing: 8) {
+            Text(.init("insufficient_storage::error::content::first".localized))
+            Text(.init("insufficient_storage::error::content::second".localized(0.021)))
+                .foregroundColor(Color.LL.Button.Warning.background)
+            Text(.init("insufficient_storage::error::content::third".localized))
+                .padding(.top, 8)
+        }
+        .padding(.vertical, 8)
+    }
+    
+    EmptyView()
+        .customAlertView(
+            isPresented: .constant(true),
+            title: .init("insufficient_storage::error::title".localized),
+            customContentView: AnyView(customContentView()),
+            buttons: [
+                AlertView.ButtonItem(type: .secondaryAction, title: "Deposit", action: {}),
+                AlertView.ButtonItem(type: .primaryAction, title: "Buy FLOW", action: {})
```

Comment:
Use localized string

Reply:
This is just for the preview, do we have to apply localization also in these cases?